### ISSUE TITLE
"zenfs list" command does not list files when path starts with ./" 

### ISSUE THREAD

**Issue by aravind-wdc** [westerndigitalcorporation/zenfs#170]
```
$ sudo ./bin/zenfs list --zbd=nvme0n1 --path=./.rocksdb                                                           
           0    Mar 09 2022 13:19:16            LOCK
       56525    Mar 09 2022 13:19:19            LOG.old.1646812163053548
       70558    Mar 09 2022 13:20:31            LOG
```
There are sst files in ZenFS written by MyRocks, but it is not getting listed.

**Comment by aravind-wdc** [comment]
This is due to 
https://github.com/westerndigitalcorporation/zenfs/blob/master/util/zenfs.cc#L68

**Comment by MaisenbacherD** [comment]
Related issue: https://github.com/westerndigitalcorporation/zenfs/issues/164

**Comment by yhr** [comment]
We can use c++ 17 path normalization to convert all filenames/paths coming in, to avoid problems comparing file name formats internally. 

See
https://en.cppreference.com/w/cpp/filesystem/path/lexically_normal

**Comment by aravind-wdc** [comment]
Resolved by #176
